docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1048,6 +1216,8 @@ message PkgInfo {
   string contact = 7;
   // url for more information, e.g. "http://support.p4.org/ref/p4/switch.p4_v1.0"
   string url = 8;
+  // Miscellaneous metadata, structured; a way to extend PkgInfo
```

Comment:
I'm not sure I understand the "a way to extend PkgInfo" comment. Isn't PkgInfo a unstructured annotations?

Reply:
`PkgInfo` is a first-class annotation which has some predefined elements like `name` and `version`, but also allows you to add arbitrary content via annotations to `main()`. Previously only unstructured annotations existed. This PR  adds structured annotations as well.